pkg/svcgen/params/builder.go

```diff
@@ -0,0 +1,275 @@
+package params
+
+import (
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+	"github.com/pkg/errors"
+	"github.com/serenize/snaker"
+)
+
+type Builder interface {
+	Build(path string, resName string, methodNames []string) (*Params, error)
+}
+
+func NewBuilder(rootDir cli.RootDir, protoDir, protoOutDir, serverDir string, pkgName string) Builder {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewBuilder should have comment or be unexported